.github/workflows/CI.yml

```diff
@@ -21,42 +21,44 @@ jobs:
           - 'nightly'
         os:
           - ubuntu-latest
-          - macOS-latest
+          - macos-13
```

Comment:
Why is this switched?

Reply:
To be precise, *`macos-latest`* was switched from `macos-12` to `macos-14` last month.
`macos-13` is the *latest* intel mac.
Of course, we could just keep it as `macos-latest`. Many packages actually do so.

Reply:
Lets just leave it at `macos-latest` then.